src/intro.md

```diff
@@ -11,3 +15,17 @@ To access the [Builder Mode](builder/intro.md) menu, press Left Alt
 To open/close the [Editor Mode](editor/intro.md), press Right Shift
 
 These keybinds can be adjusted in Minecraft's controls menu (Esc > Options > Controls)
+
+## Latest Update 
```

Comment:
Probably shouldn't have latest update here as it'd require constant maintenance to keep updated. People can look at the changelogs on modrinth/the website anyways if they want this information